conduit/pipeline/pipeline_test.go

```diff
@@ -914,32 +915,32 @@ func TestMetrics(t *testing.T) {
 		if strings.HasSuffix(*stat.Name, metrics.BlockImportTimeName) {
 			found++
 			// 1 hour in seconds
-			assert.Contains(t, stat.String(), "sample_count:1 sample_sum:3600")
+			assert.Regexp(t, regexp.MustCompile("sample_count:1 +sample_sum:3600"), stat.String())
```

Comment:
@winder 
For mysterious reasons, this tests started failing on me and I had to be more lenient on the assertions in order to pass.

Reply:
I would guess it's related to upgrading the prometheus versions.

Am I reading the regex correctly, that there is additional whitespace and some control character changes (quotes instead of angles?) but the data is otherwise the same?

Reply:
Yep. We now have `{` instead of `<` and unpredictable white space.